solidity/ecdsa/deploy/03_deploy_wallet_registry.ts

```diff
@@ -17,35 +17,52 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
     waitConfirmations: 1,
   })
 
-  const walletRegistry = await helpers.upgrades.deployProxy("WalletRegistry", {
-    contractName:
-      process.env.TEST_USE_STUBS_ECDSA === "true"
-        ? "WalletRegistryStub"
-        : undefined,
-    initializerArgs: [
-      EcdsaDkgValidator.address,
-      RandomBeacon.address,
-      ReimbursementPool.address,
-    ],
-    factoryOpts: {
-      signer: await ethers.getSigner(deployer),
-      libraries: {
-        EcdsaInactivity: EcdsaInactivity.address,
+  const [walletRegistry, proxyDeployment] = await helpers.upgrades.deployProxy(
+    "WalletRegistry",
+    {
+      contractName:
+        process.env.TEST_USE_STUBS_ECDSA === "true"
+          ? "WalletRegistryStub"
+          : undefined,
+      initializerArgs: [
+        EcdsaDkgValidator.address,
+        RandomBeacon.address,
+        ReimbursementPool.address,
+      ],
+      factoryOpts: {
+        signer: await ethers.getSigner(deployer),
+        libraries: {
+          EcdsaInactivity: EcdsaInactivity.address,
+        },
       },
-    },
-    proxyOpts: {
-      constructorArgs: [SortitionPool.address, TokenStaking.address],
-      unsafeAllow: ["external-library-linking"],
-      kind: "transparent",
-    },
-  })
+      proxyOpts: {
+        constructorArgs: [SortitionPool.address, TokenStaking.address],
+        unsafeAllow: ["external-library-linking"],
+        kind: "transparent",
+      },
+    }
+  )
 
   await helpers.ownable.transferOwnership(
     "EcdsaSortitionPool",
     walletRegistry.address,
     deployer
   )
 
+  if (hre.network.tags.etherscan) {
+    if (hre.network.tags.etherscan) {
+      await helpers.etherscan.verify(EcdsaInactivity)
+    }
```

Comment:
Do we need this `if` again? It's the same condition as in the line above.

```suggestion
    await helpers.etherscan.verify(EcdsaInactivity)
```